CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ about writing changes to this log.
 
 ## [Unreleased]
 
+- [TVIST1-729](https://jira.itkdev.dk/browse/TVIST1-729):
+  Allow hearing to start with any one part.
```

Comment:
Is this, i.e. “any one part”, proper English?

Reply:
Reworded.